### ISSUE TITLE
[Bug]: TalkBack does not read correctly the "Close" button from onboarding screens for existing users

### ISSUE THREAD

**Issue by AdinaPetridean** [mozilla-mobile/fenix#27253]
### Steps to reproduce

1. Turn TalkBack (or the available screen reader) on.
2. Update the Firefox app to the latest version and launch it.
3. Tap on "Close" button and notice what talkback announces.

### Expected behaviour

Talkback reads correctly "Close button"

### Actual behaviour

Talkback announces "button", without announcing what button it is  and to double tap to activate.

### Device name

Xperia 10 

### Android version

Android 12

### Firefox release type

Firefox Nightly

### Firefox version

Firefox Nightly 107.0a1 - 10/03

### Additional information

Please note this issue is also reproducible on Firefox Beta 106.0b3.
<img src="https://user-images.githubusercontent.com/108326843/193601052-b3197569-0bfc-4e0b-badb-8c7d2363b28b.JPEG" width="50%" /><img src="https://user-images.githubusercontent.com/108326843/193600047-e237df34-c92c-4170-9458-ae336dea6c99.JPEG" width="50%" />

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21710)

**Comment by AdinaPetridean** [comment]
Verified on Firefox Nighlty 107.0a1 - 10/06 with Xperia 10(Android 12).
